src/webgpu/api/validation/state/device_lost/destroy.spec.ts

```diff
@@ -901,9 +900,6 @@ Tests copyExternalImageToTexture from canvas on queue on destroyed device.
     u
       .combine('canvasType', kAllCanvasTypes)
       .combine('contextType', kValidCanvasContextIds)
-      .filter(({ contextType }) => {
-        return canCopyFromCanvasContext(contextType);
-      })
```

Comment:
Please verify this test works correctly after my change

Reply:
Checked these test cases and they can pass